vrk-kpa/suomidigi
# SUOPA-717 Liftup 50/50

To test: 
- `make fresh` (and clear caches after logging in)
- Log in and go to create a new landing page (https://suomidigi.docker.sh/node/add/landing_page)
- Give the page a title and add new paragraph called Liftup 50/50
   - Give the liftup a title, image and text.
   - Save the node and check that the layout is similar to the one on link below, except that the width of the liftup 50/50 is content wide - not screen width wide.
       - https://xd.adobe.com/view/c1b40caa-b2e0-43a5-78d8-9690a43f540c-b785/screen/2585ee1f-a81b-49a3-976f-98c0bdbcbf87/Suomidigi-ohjeet-tuki-koonti/
- Edit the node and duplicate Liftup 50/50 from the three dots
   - Alter the values and select the "Reverse" option
   - Save the node and check that the layout works on "reverse mode" as well.
- Also check that the layout works on mobile width
